{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "not found: next",
      "code": "ENOENT",
      "stack": "Error: not found: next\n    at getNotFoundError (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41455:51)\n    at Function.whichSync [as sync] (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41532:13)\n    at getCommandAbsolutePathSync (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43097:44)\n    at default_1 (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43106:32)\n    at /usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:179764:39"
    }
  },
  "2 error pnpm": {
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "prefix": "/Users/andreas/github/chronark/upstash-ui",
    "err": {
      "name": "pnpm",
      "message": "not found: next",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: not found: next\n    at getNotFoundError (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41455:51)\n    at Function.whichSync [as sync] (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41532:13)\n    at getCommandAbsolutePathSync (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43097:44)\n    at default_1 (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43106:32)\n    at /usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:179764:39"
    }
  }
}